SAP2000_oM/Fragments/IPanelAutoMesh.cs

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+using BH.oM.Base;
+using System.ComponentModel;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.oM.Adapters.SAP2000.Fragments
+{
+    [Description("Base interface for panel auto mesh settings in SAP2000. Contains the type of auto mesh and the properties common to all types.")]
+    [Unique]
+    public interface IPanelAutoMesh : IFragment
+    {
+        [Description("If this item is True, and if both points along an edge of the original area " + "object have the same local axes, the program makes the local axes for added points " + "along the edge the same as the edge end points.")]
+        bool LocalAxesOnEdge
+        {
+            get;
+            set;
+        }
```

Comment:
Not sure why all of these are on different lines?

Can we change all of them to be single line properties as the general convention. Especially when default values are assigned keeping them on multiple lines makes it really quite hard to read.

```suggestion
        bool LocalAxesOnEdge { get; set; }
```